### ISSUE TITLE
Can a masked text be inferenced?

### ISSUE THREAD

**Issue by annannacf** [deepset-ai/FARM#517]
We have fine tuned our own version of german BERT and would like to demo the fill-mask task. This is currently not possible with the Inferencer class, is there another way of doing it?

Transformers have the Pipeline class which is able to solve this task, but we need to pass a model of the class`~transformers.PreTrainedModel` or `~transformers.TFPreTrainedModel`. We tried converting our model to transformers but this is only possible with single head AdaptiveModel class, not with LanguageModel:
`model = LanguageModel.load(farm_model_dir, device="cpu")
tokenizer = Tokenizer.load(farm_model_dir, tokenizer_class = 'BertTokenizer')
transformer_model = model.convert_to_transformers()`
throws error:
![image](https://user-images.githubusercontent.com/22051871/92253827-4a935800-eed0-11ea-9d7b-e1d90b2529c3.png)
And trying to load the finetunned model as an AdaptiveModel has the problem that, when trained with a single BertLMHead:
![image](https://user-images.githubusercontent.com/22051871/92254536-559ab800-eed1-11ea-818d-fca2b1d60e3d.png)

and when trained with a BertLMHead and a NextSentenceHead:
![image](https://user-images.githubusercontent.com/22051871/92254599-6e0ad280-eed1-11ea-967a-a1535473fc0b.png)

Is there some way of filling the masks of a masked text?

**Comment by tholor** [comment]
Hey @annannacf, 

Sorry, for the late reply! I somehow missed this issue. 

You are totally right: 
- The Inferencer doesn't support the fill mask task
- model conversion to transformers would be a good way forward

Out of the 3 conversion approaches you mentioned, the second one seems most promising, but we'll need to investigate further to see if there is a quick fix / workaround. We will have a look at it in our upcoming sprint.

**Comment by Timoeller** [comment]
In the linked issue #533 there seems to be a solution for converting the models to transformers and using their pipeline for infering the masks.

**Comment by tholor** [comment]
Closing this as it should be fixed with the approach in #533. 
Feel free to re-open if not.
